show/main.py

```diff
@@ -7,18 +7,21 @@
 import re
 import subprocess
 import sys
+import commands
```

Comment:
imports are generally listed in alphabetical order.

Reply:
Sure. Will move it based on alphabetical order